_data/program_2021.yml

```
@@ -0,0 +1,48 @@
+- date: "2021-03-24"
+  dateReadable: "Day 1: Wednesday, 24<sup>th</sup> March 2021"
+  tracks:
+    - { title: "TrainTrack", color: "#90be4e" }
+    - { title: "HackTrack", color: "#03a9f4" }
+  timeSlots:
+    - {
+        startTime: "08:00",
```

Ok I changed my mind, I think it would be good to already have the timeslots in there as people may already want to see at what times OpenMR will be (since we will be sending out a save the date with a link to this page). Here we go (this is most likely not properly yml formatted, so sorry abt that, just learning how to add suggestions haha):

```suggestion
  - { startTime: "09:30", endTime: "16:30", sessionTitle: "TBD", speaker: "TBD", sessionType: "TBD", slides, }
```

---

If you use VS code and tell it to use an automatic linter, it will reformat the yml for you when you save the file.

I only do this kind of manual reformating for Matlab (and even then I have found ways to "lint" my code automatically to a large extend)